qiskit/algorithms/__init__.py

```diff
@@ -72,8 +72,8 @@
    MaximumLikelihoodAmplitudeEstimationResult
 
 
-Eigensolvers
-------------
+Eigen Solvers
+-------------
```

Comment:
In the text here below can we add something about these are pending deprecation and to see the eigen solvers in the next section below similar to what was done for Min Eig Solvers here - otherwise  as I noted when that was added its kinda confusing with both here otherwise.